tests/bot/test_utils.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from bot import utils
+
+
+class CaseInsensitiveDictTests(unittest.TestCase):
+    """Tests for the `CaseInsensitiveDict` container."""
+
+    def test_case_insensitive_key_access(self):
+        """Tests case insensitive key access and storage."""
```

Comment:
I don't see where in here the key is accessed using a different-cased version. All access is done through the same variable.

Reply:
woops. see 9e825ed